base/cvd/cuttlefish/host/commands/cvd/instances/lock/instance_lock.cpp

```diff
@@ -63,11 +63,13 @@ bool InstanceLockFile::operator<(const InstanceLockFile& other) const {
   return lock_file_ < other.lock_file_;
 }
 
-InstanceLockFileManager::InstanceLockFileManager() {}
+InstanceLockFileManager::InstanceLockFileManager(
+    std::string_view instance_locks_path)
+    : instance_locks_path_(instance_locks_path) {};
```

Comment:
nit: Creating `std::string` from `std::string_view`, requires a copy. If the constructor accepts an `std::string` parameter instead and `std::move` is used where necessary it's possible to avoid all extra copies.

Reply:
Ah, I forgot I made the ivar a string. I'll just make it a string_view too; I don't think it really needs to own the lock path string.

Reply:
That sounds even scarier. Can't the `InstanceLockFileManager` outlive the string from which the string_view was created?